### PR TITLE
[shopsys] moved translations on front to project-base

### LINKED ISSUE #1383: Only front-end translated texts should be in project-base translation files
### What is happening
When you want to translate your frontend to a different language - let's say German - you translate the messages in .po files in project-base. 

The problem is that some messages are not translated. Only one we found are those for sorting product list in category:
- TOP
- alphabetically A -> Z
- alphabetically Z -> A
- from the cheapest
- from most expensive

And do not want to translate the whole administration to have a new language for front-end.

### Expected result
Step 1: the sorting product list are moved to project-base
Step 2: only front-end texts are in project-base .po files or it is clear what to translate if you want to translate only FE.

### PR DESCRIPTION
**Pull request by s3tezsky**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Some translations from front could not be exported to project-base translations because they were used in classes of framework. This PR expose methods to project-base to be exported.
|New feature| No 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| fixes #1383  
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Review by vitek-rostislav — APPROVED** [review]
Hi, I really like the concept :ok_hand: I have added two questions, however, I do not need to see it again, you can then send the task to validation :+1:

File: upgrade/UPGRADE-v9.1.2-dev.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
[the remark this answers is not in the document]
Comment (s3tezsky) [reply upgrade/UPGRADE-v9.1.2-dev.md]: No it was not, fixed.

File: upgrade/UPGRADE-v9.1.2-dev.md [hunk 2]
```diff
@@ -30,3 +31,17 @@ There you can find links to upgrade notes for other versions too.
...
+    - following methods will become abstract:
+        - `Shopsys\FrameworkBundle\Model\Breadcrumb\ErrorPageBreadcrumbGenerator::getTranslatedBreadcrumbForNotFoundPage()`
+        - `Shopsys\FrameworkBundle\Model\Breadcrumb\ErrorPageBreadcrumbGenerator::getTranslatedBreadcrumbForErrorPage()`
+        - `Shopsys\FrameworkBundle\Model\Breadcrumb\SimpleBreadcrumbGenerator::getTranslatedBreadcrumbsByRouteNames()`
+        - `Shopsys\FrameworkBundle\Model\Cart\Watcher\CartWatcherFacade::getMessageForNoLongerAvailableExistingProduct()`
+        - `Shopsys\FrameworkBundle\Model\Cart\Watcher\CartWatcherFacade::getMessageForNoLongerAvailableProduct()`
+        - `Shopsys\FrameworkBundle\Model\Cart\Watcher\CartWatcherFacade::getMessageForChangedProduct()`
+        - `Shopsys\FrameworkBundle\Model\Product\Listing\ProductListOrderingModeForBrandFacade::getSupportedOrderingModesNamesById()`
+        - `Shopsys\FrameworkBundle\Model\Product\Listing\ProductListOrderingModeForListFacade::getSupportedOrderingModesNamesById()`
+        - `Shopsys\FrameworkBundle\Model\Product\Listing\ProductListOrderingModeForSearchFacade::getSupportedOrderingModesNamesById()`
```
[the remark this answers is not in the document]
Comment (s3tezsky) [reply upgrade/UPGRADE-v9.1.2-dev.md:45]: Nice catch, I have noticed I missed all deprecations in `LegalConditionsFacade` too. It is fixed now 👍

**Comment by vitek-rostislav** [comment]
I have also tested this locally, it works - sending to validation :+1:

**Review by sslt — APPROVED** [review]

**Pull request merged**